main.py

```diff
@@ -34,56 +34,56 @@
     has_wandb = False
 
 
-def quantize_model(model, args):
+def quantize_model(model: PreTrainedModel, args: Namespace):
     """main entry point to functions for model quantization"""
     tick = time.time()
     print("Loading data ...")
-    dataloader = get_loaders(
+    data = get_loaders(
```

Comment:
Rationale: this is not actually a dataloader, but a list of sequences; the previous name is misleading.